src/app/components/mirror/mirror.component.ts

```diff
@@ -9,12 +9,35 @@ import { Router } from '@angular/router';
 export class MirrorComponent implements OnInit {
   public width = 1080;
   public height = 810;
+  timeout: any = 0;
 
   constructor(private router: Router) {}
 
+  changePage(page: string) {
```

Comment:
think this logic can be relocated to a separate service to avoid code duplication